File: it.unibo.cautiousExplorer/app/src/main/java/it/unibo/executor/ApplMsgs.java
package it.unibo.executor;

import org.json.JSONObject;

public class ApplMsgs {
    final static String forwardMsg         = "{\"robotmove\":\"moveForward\", \"time\": 350}";
    final static String backwardMsg        = "{\"robotmove\":\"moveBackward\", \"time\": 350}";
    final static String microStepMsg       = "{\"robotmove\":\"moveForward\", \"time\": 10}";
    final static String littleBackwardMsg  = "{\"robotmove\":\"moveBackward\", \"time\": 10}";
    final static String turnLeftMsg        = "{\"robotmove\":\"turnLeft\", \"time\": 300}";
    final static String turnRightMsg       = "{\"robotmove\":\"turnRight\", \"time\": 300}";
    final static String haltMsg            = "{\"robotmove\":\"alarm\", \"time\": 20}";
    final static String goBackMsg          = "{\"goBack\":\"goBack\" }";
    final static String resumeMsg          = "{\"resume\":\"resume\" }";
    final static String forwardstepMsg     = "{\"robotmove\":\"moveForward\", \"time\": 350}";

    final static String activateId         = "activate";
    final static String activateMsg        = "{\"ID\":\"ARGS\" }".replace("ID", activateId);
    final static String executorStartId    = "executorstart";
    final static String executorstartMsg   = "{\"ID\":\"PATHTODO\" }".replace("ID", executorStartId);
    final static String endMoveId          = "endmove";
    final static String endMoveMsg         = "{\"ID\":\"ENDMOVE\" }".replace("ID", endMoveId);
    final static String executorEndId      = "executorend";
    final static String executorendokMsg   = "{\"ID\":\"ok\" }".replace("ID", executorEndId);
    final static String executorendkoMsg   = "{\"ID\":\"PATHDONE\" }".replace("ID", executorEndId);
    final static String runawyEndId       = "runawayend";
    final static String runawyEndMsg      = "{\"ID\":\"RESULT\" }".replace("ID", runawyEndId);
    final static String runawyStartId     = "runawaystart";
    final static String runawyStartMsg    = "{\"ID\":\"PATHTODO\" }".replace("ID", runawyStartId);

    final static String stepId            = "step";
    final static String stepMsg           = "{\"ID\":\"TIME\" }".replace("ID", stepId);
    final static String stepDoneId        = "stepDone";
    final static String stepDoneMsg       = "{\"ID\":\"ok\" }".replace("ID", stepDoneId);
    final static String stepFailId        = "stepFail";
    final static String stepFailMsg       = "{\"ID\":\"TIME\" }".replace("ID", stepFailId);



}

File: it.unibo.kotlinSupports/app/src/main/kotlin/it/unibo/executors/ApplMsgs.kt
package it.unibo.executors

import it.unibo.actor0.ApplMessage
import it.unibo.executors.ApplMsgs

object ApplMsgs {
    const val forwardMsg = "{\"robotmove\":\"moveForward\", \"time\": 350}"
    const val backwardMsg = "{\"robotmove\":\"moveBackward\", \"time\": 350}"
    const val microStepMsg = "{\"robotmove\":\"moveForward\", \"time\": 5}"
    const val littleBackwardMsg = "{\"robotmove\":\"moveBackward\", \"time\": 10}"
    const val turnLeftMsg = "{\"robotmove\":\"turnLeft\", \"time\": 300}"
    const val turnRightMsg = "{\"robotmove\":\"turnRight\", \"time\": 300}"
    const val haltMsg = "{\"robotmove\":\"alarm\", \"time\": 20}"
    const val goBackMsg = "{\"goBack\":\"goBack\" }"
    const val resumeMsg = "{\"resume\":\"resume\" }"
    const val forwardstepMsg = "{\"robotmove\":\"moveForward\", \"time\": 350}"

    const val activateId = "activate"
    val activateMsg = "{\"ID\":\"ARGS\" }".replace("ID", activateId)

    const val executorStartId = "executorstart"
    val executorstartMsg = "{\"ID\":\"PATHTODO\" }".replace("ID", executorStartId)
    const val executorDoneId = "executorDone"
    const val executorFailId = "executorFail"
    val executorendokMsg = "{\"ID\":\"ok\" }".replace("ID", executorDoneId)
    val executorendkoMsg = "{\"ID\":\"PATHDONE\" }".replace("ID", executorFailId)

    const val endMoveId = "endmove"
    val endMoveMsg = "{\"ID\":\"ENDMOVE\" }".replace("ID", endMoveId)
 //
    const val runawyEndId = "runawayend"
    val runawyEndMsg = "{\"ID\":\"RESULT\" }".replace("ID", runawyEndId)
    const val runawyStartId = "runawaystart"
    val runawyStartMsg = "{\"ID\":\"PATHTODO\" }".replace("ID", runawyStartId)
 //
    const val stepId = "step"
    val stepMsg = "{\"ID\":\"TIME\" }".replace("ID", stepId)
    const val stepDoneId = "stepDone"
    val stepDoneMsg = "{\"ID\":\"ok\" }".replace("ID", stepDoneId)
    const val stepFailId = "stepFail"
    val stepFailMsg = "{\"ID\":\"TIME\" }".replace("ID", stepFailId)



    val demoMsgAppl =
        ApplMessage("cmd","dispatch","demo","obs","todo","1")


}